velox/exec/MergeJoin.h

```diff
@@ -249,8 +256,8 @@ class MergeJoin : public Operator {
   /// block, we keep the subset of passing rows. However, if the filter failed
   /// on all rows in such a block, we add one of these rows back and update
   /// build-side columns to null.
-  struct LeftJoinTracker {
-    LeftJoinTracker(vector_size_t numRows, memory::MemoryPool* pool)
+  struct JoinTracker {
```

Comment:
nit: OuterJoinTracker might be more precise.

Reply:
@xumingming  JoinTracker is utilized by both left and right joins, not exclusively by full outer joins